Fix contest button alignment issues Fixes #8422

PrasannaMishra001 opened the pull request:
## Changes Made

### 1. **Dropdown Button (`contest-dropdown` slot)**
- **Before:**  
  - Button text and icon were misaligned, using `<p>` tags and inconsistent flex classes.  
- **After:**  
  - Applied `d-inline-block` to `<b-dropdown>` for proper alignment.  
  - Removed unnecessary `<p>` tags.  
  - Ensured icon and text stay on one line using flex alignment.
<img width="367" height="508" alt="Screenshot 2025-09-05 025911" src="https://github.com/user-attachments/assets/85de83c3-c060-459a-9dea-cea41d9ef4e9" />

---

### 2. **"See Details" Button (`contest-button-see-details` slot)**
- **Before:**  
  - Used `<p>` tag inside button causing misalignment.  
- **After:**  
  - Replaced `<p>` with inline text.  
  - Added `d-flex align-items-center justify-content-center` for consistent alignment.  
<img width="405" height="448" alt="Screenshot 2025-09-05 025844" src="https://github.com/user-attachments/assets/d4507e91-c4b7-473b-bba8-ae0b6c500014" />

---

### 3. **"Enter" Button (`contest-button-enter` slot)**
- **Before:**  
  - Similar issue with `<p>` tags and misalignment.  
- **After:**  
  - Removed `<p>` tags, keeping text inline.  
  - Ensured consistent flex classes for alignment.  

---

### Overall:
- **Consistent button alignment** using flex utilities.  
- **Removed unnecessary HTML tags** for cleaner structure.  
- **UI consistency across all buttons**: Dropdown, See Details, Enter.

pabo99 reviewed (commented):
The change looks great, thanks for fixing this.

While you’re updating the button styles, could you also address the following:

- Scoreboard button hover text color
Currently, buttons with the `btn-success` style have a hover text color of `#666`, which doesn’t show well against the green background. Could you standardize it so the hover text color is white, like the other buttons?

- Button sizing consistency
Is there a way to ensure both buttons have the same size, regardless of their type?

PrasannaMishra001 commented:
Hi @pabo99! 

Thanks for your feedback. I've implemented both changes:
- **Scoreboard button hover color**: Fixed the text color to remain white on hover instead of the problematic gray (#666)
- **Consistent button sizing**: Added uniform height and flex alignment across all button types

<img width="613" height="439" alt="image" src="https://github.com/user-attachments/assets/14ace009-9d63-4791-973b-7f20323cc3ae" />

I opted to keep the scoreboard button more horizontal space than the enter button as it creates better visual balance , having them identical width looked a bit awkward with the extra whitespace. I've set min/max width constraints to ensure consistency.

Let me know if these changes look good to merge!

pabo99 reviewed (approved):
Thank you, @PrasannaMishra001 !
